nagios/datadog_checks/nagios/data/conf.yaml.example

```diff
@@ -10,7 +10,7 @@ instances:
     ## @param nagios_conf - string - required
     ## Define the path to your Nagios configuration file.
     ## The Agent reads the main Nagios configuration file to get the locations of the Nagios log files it should watch.
-    ## The integration also works with Icinga. If you use Icinga, set `nagios_conf` to the location of your Icinga
+    ## The integration also works with Icinga v1. If you use Icinga v1, set `nagios_conf` to the location of your Icinga
```

Comment:
> The integration also works with Icinga v1.

Does that mean we are not supporting Icinga v2 ?

Reply:
The customer who reported the issue said Icinga v1 and Icinga v2 have completely different code-bases.

As for listing versions, from what I have see, we list supported version vs explicitly listing `we don't support vX`.